models/single_integrator.py

```diff
@@ -0,0 +1,50 @@
+from math import *
+from scipy.integrate import odeint
+from models.agent import *
+
+class RobotSingleIntegrator2D(Agent):
+
+    def __init__(self, size, velocity, k):
+        self.size = size
+        self.velocity = velocity
+        self.k = k
+
+    def model(self, q, t, u):
+        x, y = q
+        vx, vy = u
+        dxdt = vx 
+        dydt = vy 
+        return [dxdt, dydt]
+
+    def controller(self, q, qref, uref):
+        x, y = q
+        xref, yref = qref
+        vxref, vyref = uref
+        k1, k2  = self.k
+        # needs to be checked
+        xe = xref - x
+        ye = yref - y
+        vx = vxref + k1 * xe
+        vy = vyref * k2 * ye
+        return [vx, vy]
+
+    def bloating(self, n):
+        return 0
+        k1, k2 = self.k
+        if n != 0:
+            return sqrt(4 * n / k2)
+        else:
+            return sqrt(4 / k2)
```

Comment:
what does this function do?

Reply:
function is called while finding collision, but it returns 0 all the time, right ?

Reply:
I didn't see that part, I guess we should delete the dead code then. However, what is that function supposed to do and what is n?